src/screens/Reader.tsx

```diff
@@ -553,7 +553,7 @@ export function Reader() {
                 alignItems: 'center',
                 justifyContent: 'center',
                 minWidth: `calc((100vw - (100vw - 100%)) - ${navBarWidth}px)`, // 100vw = width excluding scrollbar; 100% = width including scrollbar
-                minHeight: '100vh',
+                minHeight: `calc(100vh - (100vh - 100%))`,
```

Comment:
does this work?

Reply:
It does, in the sense that `100vh` nets me that smidge of overflow and this doesn't.

Reply:
I think thats because this results in `calc(100vh - (100vh - 0))` since the parent element does not have a height set, so it takes 100% of 0 - which at the end makes this a min height of 0

Reply:
`getComputedStyle()` returned `100%` for it on the browser's Developer Tools
![image](https://github.com/user-attachments/assets/09b8ed38-351c-456a-81e8-543f7695bfa4)

Reply:
interesting, nonetheless, 100% of 0 height of the parent is still 0

Reply:
I tried changing it to both `max-height` and `height` just to see if it would "disappear" with 0 height, it didn't. Computed style for `height` was even in absolute units
![image](https://github.com/user-attachments/assets/79052454-262b-459d-850a-135e17cd79de)

Reply:
you can see it by adding `max-height` with both % and vh values, the % based value does not do anything while the vh one does shrink the pages in size

Reply:
this needs to be changed to the same fix as in `Page` (`calc(100vh - ${scrollbarHeight}px)`)

as it is now it breaks the single and double page mode while `fit to page` is disabled and the `reader width` is below 100%, in that case the page is not vertically centered anymore

Reply:
looks good